src/components/Activity/ActivityItem.js

```diff
@@ -0,0 +1,242 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonBase,
+  ButtonIcon,
+  IconCross,
+  IconCheck,
+  GU,
+  blockExplorerUrl,
+  textStyle,
+  useTheme,
+  IdentityBadge,
+} from '@aragon/ui'
+import { transformAddresses, getNetworkType } from '../../lib/web3-utils'
+import {
+  ACTIVITY_STATUS_PENDING,
+  ACTIVITY_STATUS_CONFIRMED,
+  ACTIVITY_STATUS_FAILED,
+  ACTIVITY_STATUS_TIMED_OUT,
+} from './symbols'
```

Comment:
Should we rename this `symbols` file to something more like `status` or `activityStatus`?

Reply:
Yes good idea! Done in https://github.com/aragon/court-dashboard/pull/100/commits/22e5f514d75b0acf46523d3f2d03ee2bf12e3cff